components/avatar.css

```diff
@@ -0,0 +1,67 @@
+/* helping class */
+/* to be removed when img component will be added */
+.img-responsive {
+  max-width: 100%;
+  height: auto;
+}
+
+.img-round {
+  border-radius: 50%;
+}
+
+/* avatar */
+.avatar,
+.avatar-sq,
+.avatar-text {
+  display: inline-block;
+}
+
+.avatar img,
+.avatar-sq img {
+  height: 100%;
+  object-fit: cover;
+  width: 100%;
+}
+
+.avatar,
+.avatar-text {
+  border-radius: 50%;
```

Comment:
this is being used in lot of components , so it can be a separate variable.

Reply:
Noted!